BrightID/src/actions/fetchUserInfo.ts

```diff
@@ -25,17 +26,20 @@ const fetchUserInfo = () => (dispatch: dispatch, getState: getState) => {
       if (!id) {
         throw new Error('id missing');
       }
-
       try {
         const {
           groups,
           score,
-          verifications = [],
           connections = [],
           isSponsored,
           invites,
         } = await api.getUserInfo(id);
-
+        const verifications = await api.getUserVerifications(id);
+        let incomingConns = await api.getConnections(id, 'inbound');
```

Comment:
This will cause additional traffic as we first get all connections included in `getUserInfo()` and then get half of them (the inbound connections) again with the `getConnections()` call.
This will not hurt the average user much, but might be a problem for the brightID stars.

Reply:
This is the only way we currently have for querying that from API. We can improve that in future:
https://github.com/BrightID/BrightID-Node/issues/206